darts/models/forecasting/forecasting_model.py

```diff
@@ -2902,7 +2902,12 @@ class FutureCovariatesLocalForecastingModel(LocalForecastingModel, ABC):
     All implementations must implement the :func:`_fit()` and :func:`_predict()` methods.
     """
 
-    def fit(self, series: TimeSeries, future_covariates: Optional[TimeSeries] = None):
+    def fit(
+        self,
+        series: TimeSeries,
+        future_covariates: Optional[TimeSeries] = None,
+        **fit_kwargs,
```

Comment:
The main concern I have here are the `EnsembleModel`s
In principle I see 3 possibilities of dealing with the `kwargs` in this case

1. Do nothing and ignore the kwargs
    - That could be confusing because I might have an `EnsembleModel` of a set of closely related models which all support the same keyword argument. Then it would be pretty confusing for the User to have kwargs in the function signature but not have them passed to the models
2. Pass kwargs to all of the models
    - That's pretty impractical because some of my models may accept a certain argument and others won't
3. instead define `kwargs: list[dict]` which needs to correspond to the number of models. `EnsembleModel.forecasting_model[0].fit` gets passed `kwargs[0]` and so on
    - That option makes most sense to me, but of course we won't really have a unified API in this case

@dennisbader What do you think?

Reply:
Looking through the code, there are some more models where I find it difficult to deal with the `kwargs`, namely models that are mainly implemented within darts than wrapped from another library, such as `Theta`, `FFT` or the Baseline Models. 
Let's take for example the `Theta` model. There are 1 or 2 calls to `hw.SimpleExpSmoothing` inside the `Theta.fit` functions, so we could pass the `kwargs` in those calls (`hw.SimpleExpSmoothing` seems to allow for meaningful keyword arguments), but I know too little about the `Theta` model to judge if that`s a meaningful expansion

Reply:
> The main concern I have here is the `EnsembleModel` In principle I see 3 possibilities of dealing with the `kwargs` in this case
> 
> 1. Do nothing and ignore the kwargs
>    
>    * That could be confusing because I might have an `EnsembleModel` of a set of closely related models which all support the same keyword argument. Then it would be pretty confusing for the User to have kwargs in the function signature but not have them passed to the models
> 2. Pass kwargs to all of the models
>    
>    * That's pretty impractical because some of my models may accept a certain argument and others won't
> 3. instead define `kwargs: list[dict]` which needs to correspond to the number of models. `EnsembleModel.forecasting_model[0].fit` gets passed `kwargs[0]` and so on
>    
>    * That option makes most sense to me, but of course we won't really have a unified API in this case
> 
> @dennisbader What do you think?

Yes, I see your point. We could do something like this:
- `**kwargs` should be passed only to the ensemble model itself (not the underlying `forecasting_models`). Currently this will only have an effect for `RegressionEnsembleModel`, where the ensemble model is one of Darts `RegressionModels`.
- add a new parameter like your option 3 that expects a single dict or a list of dicts (`Optional[Union[Dict[str, Any]], List[Dict[str, Any]]]`) that are passed to the forecasting models `fit()`.
  - if a single dict, pass the same to all models
  - if a list of dicts, the length must match the number of forecasting models. Pass each dict to the correspoinding fc model.

What do you think?